pkg/kubelet/volumemanager/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
@tcharding Should this be updated to `2017`?

Reply:
Good catch

Reply:
There are a bunch of these errors. 2015, 2016 etc. Will change all to 2017 and re-submit